cw-style.gemspec

```diff
@@ -28,8 +28,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "rubocop", "~> 0.89.1"
   spec.add_dependency "rubocop-rspec-focused", "= 0.0.3"
+  spec.add_dependency "rubocop", "~> 1.18.3"
```

Comment:
I think we might be able to replace https://github.com/charitywater/cw-style/blob/master/lib/unless_with_multiple_conditions.rb with https://github.com/rubocop/rubocop/issues/5388 now. 

From skimming https://github.com/rubocop/rubocop/blob/master/CHANGELOG.md

Reply:
Good catch, thanks! Did that and also ignored a couple more cops that seemed pretty ignorable to me.